addon/utils/polyfill/closest.js

```diff
@@ -0,0 +1,43 @@
+// Polyfill for 'closest' and `matches` in IE11.
+// https://github.com/jonathantneal/closest
+export default function polyfill(window) {
+  if (typeof window !== 'undefined' && typeof document !== 'undefined') {
+    (function(ElementProto) {
+      if (typeof ElementProto.matches !== 'function') {
```

Comment:
Here's what I've been using for IE11:
```
if (!Element.prototype.closest) {
    Element.prototype.closest = function(selector) {
      let element = this;

      do {
        if (element.msMatchesSelector(selector)) {
          return element;
        }

        element = element.parentElement || element.parentNode;
      } while (element !== null && element.nodeType === 1);

      return null;
    };
  }
```

Note that Ember only supports IE11, so you don't need to support older IE browsers.